patches/components-sync_bookmarks-bookmark_change_processor.cc.patch

```diff
@@ -0,0 +1,57 @@
+diff --git a/components/sync_bookmarks/bookmark_change_processor.cc b/components/sync_bookmarks/bookmark_change_processor.cc
+index 35a73a07bbcdd8aeba13acf37c6754aadb039768..ac7ebcbda6bc1de634b81be44ce98c2c3656b3ac 100644
+--- a/components/sync_bookmarks/bookmark_change_processor.cc
++++ b/components/sync_bookmarks/bookmark_change_processor.cc
+@@ -87,6 +87,7 @@ void BookmarkChangeProcessor::UpdateSyncNodeProperties(
+   bookmark_specifics.set_creation_time_us(src->date_added().ToInternalValue());
+   dst->SetBookmarkSpecifics(bookmark_specifics);
+   SetSyncNodeFavicon(src, model, dst);
++  brave_sync::AddBraveMetaInfo(src, model);
+   SetSyncNodeMetaInfo(src, dst);
+ }
+ 
+@@ -383,7 +384,7 @@ int64_t BookmarkChangeProcessor::UpdateSyncNode(
+ 
+ void BookmarkChangeProcessor::BookmarkMetaInfoChanged(
+     BookmarkModel* model, const BookmarkNode* node) {
+-  BookmarkNodeChanged(model, node);
++  // TODO(darkdh): make sure the meta info changed is what we want to sync
+ }
+ 
+ void BookmarkChangeProcessor::BookmarkNodeMoved(BookmarkModel* model,
+@@ -415,6 +416,7 @@ void BookmarkChangeProcessor::BookmarkNodeMoved(BookmarkModel* model,
+       error_handler()->OnUnrecoverableError(error);
+       return;
+     }
++    BRAVE_BOOKMARK_CHANGE_PROCESSOR_BOOKMARK_NODE_MOVED
+ 
+     if (!PlaceSyncNode(MOVE, new_parent, new_index, &trans, &sync_node,
+                        model_associator_)) {
+@@ -458,6 +460,7 @@ void BookmarkChangeProcessor::BookmarkNodeFaviconChanged(
+     return;
+   }
+ 
++  BRAVE_BOOKMARK_CHANGE_PROCESSOR_BOOKMARK_NODE_FAVICON_CHANGED
+   CreateOrUpdateSyncNode(node);
+ }
+ 
+@@ -490,6 +493,7 @@ void BookmarkChangeProcessor::BookmarkNodeChildrenReordered(
+       DCHECK_EQ(sync_child.GetParentId(),
+                 model_associator_->GetSyncIdFromChromeId(node->id()));
+ 
++      BRAVE_BOOKMARK_CHANGE_PROCESSOR_CHILDREN_REORDERED
+       if (!PlaceSyncNode(MOVE, node, i, &trans, &sync_child,
+                          model_associator_)) {
+         syncer::SyncError error(FROM_HERE,
+@@ -718,9 +722,11 @@ void BookmarkChangeProcessor::ApplyChangesFromSyncModel(
+                    << src.GetBookmarkSpecifics().url();
+         continue;
+       }
++      bookmark_model_->SetNodeMetaInfo(dst, "FirstLoadedFavicon", "true");
```

Comment:
can we move this into `BRAVE_BOOKMARK_CHANGE_PROCESSOR_APPLY_CHANGES_FROM_SYNC_MODEL`?

Reply:
addressed in 3b1d457dc14d5a36353c45699459d728c5cb08ef